src/bin/genemichaels.rs

```diff
@@ -82,14 +91,18 @@ impl FromStr for On {
     }
 }
 
-#[derive(Parser)]
+#[derive(Parser, Clone)]
 struct Args {
     #[arg(help = "Files to format in place; if none specified formats stdin and writes result to stdout")]
     files: Vec<PathBuf>,
-    #[arg(short, long, default_value_t = FormatConfig::default().max_width)]
-    line_length: usize,
     #[arg(short, long, help = "Won't emit any output")]
     quiet: bool,
+    #[arg(short, long, help = "Formats the entire project")]
```

Comment:
Would you mention that it uses `Cargo.toml` here? I just checked, and I guess probably `package` is the most accurate term: https://doc.rust-lang.org/cargo/appendix/glossary.html#package

Reply:
Good call